README.md

```diff
@@ -96,7 +70,7 @@ REMOVE - *Example(Create a calculator)*
 - Head over to [Hashnode](https://hashnode.com/).
 - Create an account if you don't have one
 - Create a new blog post.
-- Using markdown write up the steps you took to create a project. Using links, libraries, images etc.
+- Using markdown, write up the steps you took to create a project. I was using links, libraries, images, etc.
```

Comment:
"I was using" doesn't work here. 

Use the below instead.

Using markdown, write up the steps you took to create a project, including any links, libraries, images, etc.

Reply:
noted